static/src/stylesheets/module/commercial/glabs/_hosted.scss

```diff
@@ -196,38 +196,53 @@ $renaultLight: #fff9e9;
         opacity: 0;
     }
 
-    &.vjs-paused {
-        .vjs-fullscreen-control,
-        .vjs-volume-menu-button,
-        .vjs-progress-control {
-            @include hosted-fade-out;
-        }
+    &.vjs-playing .vjs-time-control,
+    .vjs-fullscreen-control,
+    .vjs-volume-menu-button,
+    .vjs-progress-control {
+        @include hosted-fade-out;
+        transition-delay: 1500ms;
+    }
 
-        .vjs-big-play-button {
-            @include hosted-fade-in;
-        }
+    .vjs-big-play-button {
+        @include hosted-fade-in;
+    }
 
-        .vjs-control-bar {
-            background-color: unset;
-            transition: background-color 1s linear;
-        }
+    .vjs-control-bar {
+        background-color: unset;
+        transition: background-color 1s linear;
+        transition-delay: 1500ms;
```

Comment:
You can simply append it to `transition`, as the second value parsed as a duration is assigned to `transition-delay`.